src-electron/generator/helper-session.js

```diff
@@ -612,18 +616,22 @@ function all_user_cluster_commands_irrespective_of_manufaturing_specification(
 
 /**
  * Creates endpoint type cluster attribute iterator. This fetches all
- * manufacturing and non-manufaturing specific attributes which have been enabled
- * on added endpoints
+ * manufacturing and standard attributes which have been enabled on
+ * added endpoints based on the name and side of the cluster. When side
+ * is not mentioned then client and server attributes are returned.
+ * Available Options:
+ * - removeKey: Removes one of the keys from the map(for eg keys in db-mapping.js)
```

Comment:
Why is this useful?  And why is it useful to only allow removing one key?

Reply:
This was added as an option to handle https://github.com/project-chip/zap/blob/master/src-electron/generator/matter/app/zap-templates/common/ClustersHelper.js#L287 when replacing `chip_server_cluster_attributes` and generate the same code using `enabled_attributes_for_cluster_and_side` and `zcl_attributes_server`

Reply:
I see your point though. I can update it to removeMultiple keys.

Reply:
I see, ok.  Yeah, having some sort of more flexible setup here, if it's not too hard to implement and not too hard to then use, would be good.

Reply:
Fixed this